quickstart-tracing.yml

```diff
@@ -40,7 +40,9 @@ services:
       # - ELASTIC_APM_SERVICE_NAME="ORY Hydra"
       # - ELASTIC_APM_SERVICE_VERSION="1.9.0"
       # - ELASTIC_APM_ENVIRONMENT="devel"
-
+      ### Opentelemetry ###
+      ### See env vars here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md ###
```

Comment:
This file is autogenerated, so it will be overwritten by the next automated run :)

You can change the content of this by updating the "description" in the JSON Schema!

Reply:
Ohhh, my bad, I mistook this file for another file! This is perfect, thank you :)